base/ltcmd.dtx

```diff
@@ -3073,12 +3263,48 @@
 %   prevents loss of braces, and there is then a test to see if there are
 %   nested delimiters to handle.
 %    \begin{macrocode}
-\cs_new_protected:Npn \@@_grab_D_aux:NNnNNN #1#2#3#4#5#6
+\group_begin:
+  \char_set_catcode_other:N \^^M
+  \cs_new_protected:Npn \@@_grab_D_aux:NNnNNNN #1#2#3#4#5#6#7
+    {
+      \@@_grab_D_aux:NNnNN #1#2 {#3} #4 #6
+      #7
+        {
+          \group_begin:
+            \@@_grab_D_verb_safe:N #1
+        }
+      #5 #1
```

Comment:
Again, I think we need a decision on first line handling before adjusting here, so will leave open for the record.

Reply:
Might be enough to document the technical issue, but perhaps with a bit more information than just saying "for technical reasons". More on the line  `o f` and the like is not a good idea, because it means ... in such and such ... situations, so better always use ...